### DISCUSSION TITLE
Is there a per-option comparison table for differences with self hosting?

### DISCUSSION THREAD

**Discussion by sunecosuri** [renovatebot/renovate#8308] [Request Help]
Is there a per-option comparison table for differences with self hosting?
I would like to know more about what features are available and what features are not available.
I might be able to contribute a bit to the documentation if it is somewhere. Thank you.

**Comment by viceice** [comment]
* Self-host bot options: https://docs.renovatebot.com/self-hosted-configuration/
* Normal options: https://docs.renovatebot.com/configuration-options/

**Comment by rarkins** [comment]
Do you mean installing from npm vs running the slim docker image vs running the full docker image?
